WordPress/gutenberg
# RNMobile Add size options to mobile  image block

## Description

This PR covers adding image size options to the gutenberg mobile editor as specified in [gb-mobile issue 1263](https://github.com/wordpress-mobile/gutenberg-mobile/issues/1329)
## How has this been tested?



Tested using iPhone X simulator running iOS 12.4 and Samsung S10 running API 28. Steps used for testing:
1.  Building Example app with metro bundler running with [gutenberg-mobile master branch 0e597a4](https://github.com/wordpress-mobile/gutenberg-mobile/commit/0e597a4afb8c7dfbcb4da0ca943d73102941f52b)
2. Click on an Image block (that already has an image)
3. Press the gear button to bring up Image Options menu
4. Verify new Size option exists, click size options
5. Verify size options are shown and choose one.
6. Re-open Image options menu and verify previously selected size is now shown as selected.

For verifying changes are correctly submitted to web backend:
1. Run the metro bundler for this branch running in gutenberg-mobile and build either Android or iOS WordPress app
2. Use a wordpress.com account and navigate to a post, and begin editing in Gutenberg editor.
3. Follow steps above to update Image Size options. Click Update.
4. One the web, log in to the wordpress.com account, navigate to the article, and verify that the new Image size setting is shown.

## Screenshots 
![size-options](https://user-images.githubusercontent.com/1103838/63907538-b0e9be80-c9d0-11e9-83f4-82e2aeb3759a.gif)

### Utilizes ActionSheet on iOS:
![image](https://user-images.githubusercontent.com/1103838/63906931-66ffd900-c9ce-11e9-9a33-2c65d925e382.png)


## Types of changes




This code introduces the ability to update Image Size Options from the Gutenberg editor, the same way you can in the Aztec Editor, and in Gutenberg on the web. 

## Checklist:
